Formula/firebase-cli.rb

```diff
@@ -1,31 +0,0 @@
-require "language/node"
```

Comment:
Please don't remove other formula

Reply:
Is was not aware of the that change, not sure where originated. was not intended at all.